main/src/cgeo/geocaching/CacheListActivity.java

```diff
@@ -1536,9 +1536,9 @@ public void onClick(final View arg0) {
             showProgress(true);
             showFooterLoadingCaches();
 
-            getSupportLoaderManager().restartLoader(CacheListLoaderType.NEXT_PAGE.getLoaderId(), null, CacheListActivity.this);
+            LoaderManager.getInstance(CacheListActivity.this).restartLoader(CacheListLoaderType.NEXT_PAGE.getLoaderId(), null, CacheListActivity.this);
             // the loader for subsequent pages takes over - therefore the initial loader needs to be destroyed
-            getSupportLoaderManager().destroyLoader(type.getLoaderId());
+            LoaderManager.getInstance(CacheListActivity.this).destroyLoader(type.getLoaderId());
```

Comment:
Why not use `LoaderManager.getInstance(this).` as with the other changes?

Reply:
See above.